### PR TITLE
Improvements to osc new-app and osc help to show cleaner info

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
@csrwng review please

File: pkg/cmd/server/start.go [hunk 1]
```diff
@@ -165,7 +166,7 @@ func NewCommandStartServer(name string) *cobra.Command {
 	flag.Var(&cfg.KubernetesPublicAddr, "public-kubernetes", "The Kubernetes server address for use by public clients, if different. (host, host:port, or URL). Defaults to same as --kubernetes.")
 	flag.Var(&cfg.PortalNet, "portal-net", "A CIDR notation IP range from which to assign portal IPs. This must not overlap with any IP ranges assigned to nodes for pods.")
 
-	flag.StringVar(&cfg.ImageFormat, "images", "openshift/origin-${component}:${version}", "When fetching images used by the cluster for important components, use this format. By default, the master will use the latest release.")
+	flag.StringVar(&cfg.ImageFormat, "images", "openshift/origin-${component}:${version}", "When fetching images used by the cluster for important components, use this format on both master and nodes. Will use the latest release by default.")
```
Comment (csrwng) [inline pkg/cmd/server/start.go]: > Will use the latest release by default

Maybe "The latest release will be used by default" is a better structured sentence?

File: Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/update.go [hunk 2]
```diff
@@ -115,7 +118,7 @@ func updateWithPatch(cmd *cobra.Command, args []string, f *Factory, patch string
 	data, err := helper.Codec.Encode(obj)
 	checkErr(err)
 
-	err = helper.Update(namespace, name, true, data)
+	obj, err = helper.Update(namespace, name, true, data)
```
Comment (csrwng) [inline Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/update.go]: It doesn't look like obj is used after this, shouldn't it be _ instead?

**Comment by csrwng** [comment]
LGTM - really minor stuff above.

File: Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/update.go (hunk 2 above)
Comment (smarterclayton) [inline Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/update.go]: Eh... if we ever return it, we should. 

----- Original Message -----

File: pkg/cmd/server/start.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/cmd/server/start.go]: Fixed.

----- Original Message -----

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**